pkg/mem/helpers.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * Minio Cloud Storage, (C) 2016, 2017, 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mem
+
+import (
+	"fmt"
+)
+
+const (
+	bytes     = 1
+	kiloBytes = bytes << 10
+	megaBytes = kiloBytes << 10
+	gigaBytes = megaBytes << 10
+	teraBytes = gigaBytes << 10
+)
+
+func fmtBytes(bytes uint64) string {
```

Comment:
you won't need this you can use `humanize.IBytes()` - we already depend on this external package. We use IEC size. internally.